index.bs

```diff
@@ -2467,6 +2468,79 @@ object=] for a given credential. It has the following format:
   NOTE: The names in the Name column in the above table are only for reference within this document, and are not present in the
   actual representation of the [=attested credential data=].
 
+#### Examples of `credentialPublicKey` Values encoded in COSE_Key format #### {#sctn-encoded-credPubKey-examples}
+
+This section provides examples of COSE_Key-encoded Eliptic Curve and RSA public keys for the ES256, PS256, and RS256
+signature algorithms. These examples adhere to the rules defined above for the [=credentialPublicKey=] value, and are presented in [[CDDL]] for clarity. 
+
+[[RFC8152]] [Section 7](https://tools.ietf.org/html/rfc8152#section-7) defines the general framework for all
+COSE_Key-encoded keys.
+Specific key types for specific algorithms are defined in other sections of [[RFC8152]] as well as in other specifications,
+as noted below.
+
+Below is an example of a COSE_Key-encoded Eliptic Curve public key in EC2 format (see [[RFC8152]]
+[Section 13.1](https://tools.ietf.org/html/rfc8152#section-13.1)), on the P-256 curve, to be used with the ES256 signature
+algorithm (ECDSA w/ SHA-256, see [[RFC8152]] [Section 8.1](https://tools.ietf.org/html/rfc8152#section-8.1)):
+
+<pre class="example" highlight="json">
+  {
+     1: 2,  ; kty: EC2 key type
+     3:-7,  ; alg: ES256 signature algorithm
+    -1: 1,  ; crv: P-256 curve
+    -2: x,  ; x-coordinate as byte string 32 bytes in length
+              e.g., in hex: 65eda5a12577c2bae829437fe338701a10aaa375e1bb5b5de108de439c08551d
+    -3: y   ; y-coordinate as byte string 32 bytes in length
+              e.g., in hex: 1e52ed75701163f7f9e40ddf9f341b3dc9ba860af7e0ca7ca7e9eecd0084d19c
+  }
+</pre>
+
+Below is the above Eliptic Curve public key encoded in the [=CTAP canonical CBOR encoding form=], whitespace and line breaks
+are included here for clarity and to match the [[CDDL]] presentation above:
+
+<pre class="example" highlight="json">
+  A5
+     01  02
+
+     03  26
+
+     20  01
+
+     21  58 20   65eda5a12577c2bae829437fe338701a10aaa375e1bb5b5de108de439c08551d
+
+     22  58 20   1e52ed75701163f7f9e40ddf9f341b3dc9ba860af7e0ca7ca7e9eecd0084d19c
+</pre>
+
+
+Below is an example of a COSE_Key-encoded 2048-bit RSA public key (see [[RFC8230]] [Section 4](https://tools.ietf.org/html/rfc8230#section-4))
+to be used with the RSASSA-PSS w/ SHA-256 signature algorithm 
+(see [[RFC8230]] [Section 2](https://tools.ietf.org/html/rfc8230#section-2)):
+
+<pre class="example" highlight="json">
+  {
+     1:  3,  ; kty: RSA key type
+     3:-37,  ; alg: RS256
+    -1:  n,  ; n:   RSA modulus n byte string 257 bytes in length
+                    e.g., in hex (middle bytes elided for brevity): DB5F651550...6DC6548ACC3
+    -2:  e   ; e:   RSA public exponent e byte string 3 bytes in length
+                    e.g., in hex: 010001
+  }
+</pre>
+
+Below is an example of a COSE_Key-encoded 2048-bit RSA public key (see [[RFC8230]]
```

Comment:
(Could also just say that it's the same key because it's also RSA. Your call.)

Reply:
done